{"uid":"32047c9fba577c0d","name":"Delete User Account","fullName":"Flask.tests_Selenium.test_userflow_ui_selenium#test_delete_account","historyId":"6500a03c9eb66e4d1166fc36aa01dab7","time":{"start":1760732516198,"stop":1760732516313,"duration":115},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=141.0.7390.107)\nStacktrace:\n#0 0x5603c006967a <unknown>\n#1 0x5603bfae8566 <unknown>\n#2 0x5603bfadf470 <unknown>\n#3 0x5603bfad0926 <unknown>\n#4 0x5603bfad272b <unknown>\n#5 0x5603bfad0eb1 <unknown>\n#6 0x5603bfad0650 <unknown>\n#7 0x5603bfad0302 <unknown>\n#8 0x5603bface142 <unknown>\n#9 0x5603bface92a <unknown>\n#10 0x5603bfaeba99 <unknown>\n#11 0x5603bfb86ee5 <unknown>\n#12 0x5603bfb60632 <unknown>\n#13 0x5603bfb86328 <unknown>\n#14 0x5603bfb60403 <unknown>\n#15 0x5603bfb2cb02 <unknown>\n#16 0x5603bfb2d7c1 <unknown>\n#17 0x5603c002d448 <unknown>\n#18 0x5603c00312af <unknown>\n#19 0x5603c00148d9 <unknown>\n#20 0x5603c0031e55 <unknown>\n#21 0x5603bfffa13f <unknown>\n#22 0x5603c00564b8 <unknown>\n#23 0x5603c0056693 <unknown>\n#24 0x5603c0068613 <unknown>\n#25 0x7f1036c9caa4 <unknown>\n#26 0x7f1036d29c6c <unknown>","statusTrace":"driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"6f016f8b381bda644151c2534041539e\")>\n\n    @allure.severity(Severity.CRITICAL)\n    @allure.title(\"Delete User Account\")\n    def test_delete_account(driver):\n>       test_login(driver)\n\nFlask/tests_Selenium/test_userflow_ui_selenium.py:127: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nFlask/tests_Selenium/test_userflow_ui_selenium.py:60: in test_login\n    driver.get(f\"{BASE_URL}/login\")\n/opt/hostedtoolcache/Python/3.10.19/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:479: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.10.19/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:454: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f29aaaa9990>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#24 0x5603c0068613 \\\\u003Cunknown>\\\\n#25 0x7f1036c9caa4 \\\\u003Cunknown>\\\\n#26 0x7f1036d29c6c \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                try:\n                    value = json.loads(value_json)\n                    if isinstance(value, dict):\n                        if len(value) == 1:\n                            value = value[\"value\"]\n                        status = value.get(\"error\", None)\n                        if not status:\n                            status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                            message = value.get(\"value\") or value.get(\"message\")\n                            if not isinstance(message, str):\n                                value = message\n                                message = message.get(\"message\")\n                        else:\n                            message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=141.0.7390.107)\nE       Stacktrace:\nE       #0 0x5603c006967a <unknown>\nE       #1 0x5603bfae8566 <unknown>\nE       #2 0x5603bfadf470 <unknown>\nE       #3 0x5603bfad0926 <unknown>\nE       #4 0x5603bfad272b <unknown>\nE       #5 0x5603bfad0eb1 <unknown>\nE       #6 0x5603bfad0650 <unknown>\nE       #7 0x5603bfad0302 <unknown>\nE       #8 0x5603bface142 <unknown>\nE       #9 0x5603bface92a <unknown>\nE       #10 0x5603bfaeba99 <unknown>\nE       #11 0x5603bfb86ee5 <unknown>\nE       #12 0x5603bfb60632 <unknown>\nE       #13 0x5603bfb86328 <unknown>\nE       #14 0x5603bfb60403 <unknown>\nE       #15 0x5603bfb2cb02 <unknown>\nE       #16 0x5603bfb2d7c1 <unknown>\nE       #17 0x5603c002d448 <unknown>\nE       #18 0x5603c00312af <unknown>\nE       #19 0x5603c00148d9 <unknown>\nE       #20 0x5603c0031e55 <unknown>\nE       #21 0x5603bfffa13f <unknown>\nE       #22 0x5603c00564b8 <unknown>\nE       #23 0x5603c0056693 <unknown>\nE       #24 0x5603c0068613 <unknown>\nE       #25 0x7f1036c9caa4 <unknown>\nE       #26 0x7f1036d29c6c <unknown>\n\n/opt/hostedtoolcache/Python/3.10.19/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:232: WebDriverException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1760732515948,"stop":1760732516197,"duration":249},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=141.0.7390.107)\nStacktrace:\n#0 0x5603c006967a <unknown>\n#1 0x5603bfae8566 <unknown>\n#2 0x5603bfadf470 <unknown>\n#3 0x5603bfad0926 <unknown>\n#4 0x5603bfad272b <unknown>\n#5 0x5603bfad0eb1 <unknown>\n#6 0x5603bfad0650 <unknown>\n#7 0x5603bfad0302 <unknown>\n#8 0x5603bface142 <unknown>\n#9 0x5603bface92a <unknown>\n#10 0x5603bfaeba99 <unknown>\n#11 0x5603bfb86ee5 <unknown>\n#12 0x5603bfb60632 <unknown>\n#13 0x5603bfb86328 <unknown>\n#14 0x5603bfb60403 <unknown>\n#15 0x5603bfb2cb02 <unknown>\n#16 0x5603bfb2d7c1 <unknown>\n#17 0x5603c002d448 <unknown>\n#18 0x5603c00312af <unknown>\n#19 0x5603c00148d9 <unknown>\n#20 0x5603c0031e55 <unknown>\n#21 0x5603bfffa13f <unknown>\n#22 0x5603c00564b8 <unknown>\n#23 0x5603c0056693 <unknown>\n#24 0x5603c0068613 <unknown>\n#25 0x7f1036c9caa4 <unknown>\n#26 0x7f1036d29c6c <unknown>","statusTrace":"driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"6f016f8b381bda644151c2534041539e\")>\n\n    @allure.severity(Severity.CRITICAL)\n    @allure.title(\"Delete User Account\")\n    def test_delete_account(driver):\n>       test_login(driver)\n\nFlask/tests_Selenium/test_userflow_ui_selenium.py:127: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nFlask/tests_Selenium/test_userflow_ui_selenium.py:60: in test_login\n    driver.get(f\"{BASE_URL}/login\")\n/opt/hostedtoolcache/Python/3.10.19/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:479: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.10.19/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:454: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f29aaaa9990>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#24 0x5603c0068613 \\\\u003Cunknown>\\\\n#25 0x7f1036c9caa4 \\\\u003Cunknown>\\\\n#26 0x7f1036d29c6c \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                try:\n                    value = json.loads(value_json)\n                    if isinstance(value, dict):\n                        if len(value) == 1:\n                            value = value[\"value\"]\n                        status = value.get(\"error\", None)\n                        if not status:\n                            status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                            message = value.get(\"value\") or value.get(\"message\")\n                            if not isinstance(message, str):\n                                value = message\n                                message = message.get(\"message\")\n                        else:\n                            message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=141.0.7390.107)\nE       Stacktrace:\nE       #0 0x5603c006967a <unknown>\nE       #1 0x5603bfae8566 <unknown>\nE       #2 0x5603bfadf470 <unknown>\nE       #3 0x5603bfad0926 <unknown>\nE       #4 0x5603bfad272b <unknown>\nE       #5 0x5603bfad0eb1 <unknown>\nE       #6 0x5603bfad0650 <unknown>\nE       #7 0x5603bfad0302 <unknown>\nE       #8 0x5603bface142 <unknown>\nE       #9 0x5603bface92a <unknown>\nE       #10 0x5603bfaeba99 <unknown>\nE       #11 0x5603bfb86ee5 <unknown>\nE       #12 0x5603bfb60632 <unknown>\nE       #13 0x5603bfb86328 <unknown>\nE       #14 0x5603bfb60403 <unknown>\nE       #15 0x5603bfb2cb02 <unknown>\nE       #16 0x5603bfb2d7c1 <unknown>\nE       #17 0x5603c002d448 <unknown>\nE       #18 0x5603c00312af <unknown>\nE       #19 0x5603c00148d9 <unknown>\nE       #20 0x5603c0031e55 <unknown>\nE       #21 0x5603bfffa13f <unknown>\nE       #22 0x5603c00564b8 <unknown>\nE       #23 0x5603c0056693 <unknown>\nE       #24 0x5603c0068613 <unknown>\nE       #25 0x7f1036c9caa4 <unknown>\nE       #26 0x7f1036d29c6c <unknown>\n\n/opt/hostedtoolcache/Python/3.10.19/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:232: WebDriverException","steps":[{"name":"Open Login Page","time":{"start":1760732516198,"stop":1760732516313,"duration":115},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=141.0.7390.107)\nStacktrace:\n#0 0x5603c006967a <unknown>\n#1 0x5603bfae8566 <unknown>\n#2 0x5603bfadf470 <unknown>\n#3 0x5603bfad0926 <unknown>\n#4 0x5603bfad272b <unknown>\n#5 0x5603bfad0eb1 <unknown>\n#6 0x5603bfad0650 <unknown>\n#7 0x5603bfad0302 <unknown>\n#8 0x5603bface142 <unknown>\n#9 0x5603bface92a <unknown>\n#10 0x5603bfaeba99 <unknown>\n#11 0x5603bfb86ee5 <unknown>\n#12 0x5603bfb60632 <unknown>\n#13 0x5603bfb86328 <unknown>\n#14 0x5603bfb60403 <unknown>\n#15 0x5603bfb2cb02 <unknown>\n#16 0x5603bfb2d7c1 <unknown>\n#17 0x5603c002d448 <unknown>\n#18 0x5603c00312af <unknown>\n#19 0x5603c00148d9 <unknown>\n#20 0x5603c0031e55 <unknown>\n#21 0x5603bfffa13f <unknown>\n#22 0x5603c00564b8 <unknown>\n#23 0x5603c0056693 <unknown>\n#24 0x5603c0068613 <unknown>\n#25 0x7f1036c9caa4 <unknown>\n#26 0x7f1036d29c6c <unknown>\n\n","statusTrace":"  File \"/home/runner/work/GithubPortfolio/GithubPortfolio/Flask/tests_Selenium/test_userflow_ui_selenium.py\", line 60, in test_login\n    driver.get(f\"{BASE_URL}/login\")\n  File \"/opt/hostedtoolcache/Python/3.10.19/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 479, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.10.19/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 454, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.10.19/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"27979e1c53ced9e","name":"Screenshot","source":"27979e1c53ced9e.png","type":"image/png","size":30008}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1760732516406,"stop":1760732516465,"duration":59},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"severity","value":"critical"},{"name":"parentSuite","value":"Flask.tests_Selenium"},{"name":"suite","value":"test_userflow_ui_selenium"},{"name":"host","value":"runnervmzdgdc"},{"name":"thread","value":"2945-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"Flask.tests_Selenium.test_userflow_ui_selenium"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":4,"skipped":0,"passed":0,"unknown":0,"total":4},"items":[{"uid":"6cd233c654758fec","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=141.0.7390.107)\nStacktrace:\n#0 0x56442e31067a <unknown>\n#1 0x56442dd8f566 <unknown>\n#2 0x56442dd86470 <unknown>\n#3 0x56442dd77926 <unknown>\n#4 0x56442dd7972b <unknown>\n#5 0x56442dd77eb1 <unknown>\n#6 0x56442dd77650 <unknown>\n#7 0x56442dd77302 <unknown>\n#8 0x56442dd75142 <unknown>\n#9 0x56442dd7592a <unknown>\n#10 0x56442dd92a99 <unknown>\n#11 0x56442de2dee5 <unknown>\n#12 0x56442de07632 <unknown>\n#13 0x56442de2d328 <unknown>\n#14 0x56442de07403 <unknown>\n#15 0x56442ddd3b02 <unknown>\n#16 0x56442ddd47c1 <unknown>\n#17 0x56442e2d4448 <unknown>\n#18 0x56442e2d82af <unknown>\n#19 0x56442e2bb8d9 <unknown>\n#20 0x56442e2d8e55 <unknown>\n#21 0x56442e2a113f <unknown>\n#22 0x56442e2fd4b8 <unknown>\n#23 0x56442e2fd693 <unknown>\n#24 0x56442e30f613 <unknown>\n#25 0x7fac3689caa4 <unknown>\n#26 0x7fac36929c6c <unknown>","time":{"start":1760731961599,"stop":1760731961719,"duration":120}},{"uid":"1d7b14126f7a43cf","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=141.0.7390.107)\nStacktrace:\n#0 0x55a6e034967a <unknown>\n#1 0x55a6dfdc8566 <unknown>\n#2 0x55a6dfdbf470 <unknown>\n#3 0x55a6dfdb0926 <unknown>\n#4 0x55a6dfdb272b <unknown>\n#5 0x55a6dfdb0eb1 <unknown>\n#6 0x55a6dfdb0650 <unknown>\n#7 0x55a6dfdb0302 <unknown>\n#8 0x55a6dfdae142 <unknown>\n#9 0x55a6dfdae92a <unknown>\n#10 0x55a6dfdcba99 <unknown>\n#11 0x55a6dfe66ee5 <unknown>\n#12 0x55a6dfe40632 <unknown>\n#13 0x55a6dfe66328 <unknown>\n#14 0x55a6dfe40403 <unknown>\n#15 0x55a6dfe0cb02 <unknown>\n#16 0x55a6dfe0d7c1 <unknown>\n#17 0x55a6e030d448 <unknown>\n#18 0x55a6e03112af <unknown>\n#19 0x55a6e02f48d9 <unknown>\n#20 0x55a6e0311e55 <unknown>\n#21 0x55a6e02da13f <unknown>\n#22 0x55a6e03364b8 <unknown>\n#23 0x55a6e0336693 <unknown>\n#24 0x55a6e0348613 <unknown>\n#25 0x7f6ad309caa4 <unknown>\n#26 0x7f6ad3129c6c <unknown>","time":{"start":1760731616218,"stop":1760731616332,"duration":114}},{"uid":"f638df9eeb444dcb","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=141.0.7390.107)\nStacktrace:\n#0 0x561cb5b6667a <unknown>\n#1 0x561cb55e5566 <unknown>\n#2 0x561cb55dc470 <unknown>\n#3 0x561cb55cd926 <unknown>\n#4 0x561cb55cf72b <unknown>\n#5 0x561cb55cdeb1 <unknown>\n#6 0x561cb55cd650 <unknown>\n#7 0x561cb55cd302 <unknown>\n#8 0x561cb55cb142 <unknown>\n#9 0x561cb55cb92a <unknown>\n#10 0x561cb55e8a99 <unknown>\n#11 0x561cb5683ee5 <unknown>\n#12 0x561cb565d632 <unknown>\n#13 0x561cb5683328 <unknown>\n#14 0x561cb565d403 <unknown>\n#15 0x561cb5629b02 <unknown>\n#16 0x561cb562a7c1 <unknown>\n#17 0x561cb5b2a448 <unknown>\n#18 0x561cb5b2e2af <unknown>\n#19 0x561cb5b118d9 <unknown>\n#20 0x561cb5b2ee55 <unknown>\n#21 0x561cb5af713f <unknown>\n#22 0x561cb5b534b8 <unknown>\n#23 0x561cb5b53693 <unknown>\n#24 0x561cb5b65613 <unknown>\n#25 0x7f885d09caa4 <unknown>\n#26 0x7f885d129c6c <unknown>","time":{"start":1760730794149,"stop":1760730794241,"duration":92}}]},"tags":[]},"source":"32047c9fba577c0d.json","parameterValues":[]}